<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OS2Class</class>
 <widget class="QMainWindow" name="OS2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>452</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>OS2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="logList">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>601</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>页面调度日志</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FIFOButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FIFO</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LRUButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LRU</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>请选择置换算法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择指令条数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择页面大小</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择分配的内存块数量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="singleButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>单指令执行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>全部执行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>缺页数量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>350</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>缺页率：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="insNum">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>320</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="pageSize">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="frameNum">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>220</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="pageErrorNum">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>350</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pageErrorRate">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateInsSeq">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>产生指令访问序列</string>
    </property>
   </widget>
   <widget class="QListWidget" name="insIndex">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>指令序列</string>
    </property>
   </widget>
   <widget class="QListWidget" name="insLogicalAddr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>61</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>序号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>逻辑地址</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="os2.qrc"/>
 </resources>
 <connections/>
</ui>
